frontend/src/components/AuthValidationErrors.tsx

```diff
@@ -0,0 +1,50 @@
+import { TiInfo } from 'react-icons/ti'
+import { IoClose } from 'react-icons/io5'
+import { classNames } from '~/utils/classNames'
+import React from 'react'
+
+interface Props {
+  errors: any
+  setErrors?: any
+  className?: string
+}
+
+const AuthValidationErrors: React.FC<Props> = (props): JSX.Element => { 
+  const { errors, className, setErrors } = props
+
+  const handleClose = (): void => setErrors([])
+
+  return (
+    <React.Fragment>
```

Comment:
you can replace <React.Fragment> with just <>
```suggestion
    <>
```